code/modules/mob/living/simple_animal/hostile/terror_spiders/terror_ai.dm

```diff
@@ -0,0 +1,310 @@
+// --------------------------------------------------------------------------------
+// --------------------- TERROR SPIDERS: TARGETING CODE -----------------------
+// --------------------------------------------------------------------------------
+
+/mob/living/simple_animal/hostile/poison/terror_spider/ListTargets()
+	var/list/targets1 = list()
```

Comment:
To prioritize targets. The idea being that poisonous spiders go after injectable targets first. If they can't find any, they then go after normal human targets. If they can't find any of those, only at that point do they attack machines. It tries to ensure that the spiders don't go after targets they cannot hurt much when better targets are available.